CMakeLists.txt

```diff
@@ -3,9 +3,11 @@ project(module_a)
 
 set(CMAKE_C_STANDARD 99)
 
-add_library(module_a src/module_a.c)
-target_include_directories(module_a PUBLIC include)
-
+add_library(module_a SHARED src/module_a.c)
```

Comment:
The SHARED was missing, CMake by default create STATIC libraries

Reply:
ok that's good to know, im far from a cmake expert